platform/local_country_file_utils.hpp

```diff
@@ -18,7 +18,11 @@ namespace migrate
 }
 
 // Removes all files downloader creates during downloading of a country.
+// Note. The the maps are deleted from writable dir/|dataDir|/|version| directory.
+// If |dataDir| is empty (or is not set) the function delete maps from writable dir/|dataDir|/.
```

Comment:
// If |dataDir| is empty (or is not set) the function delete maps from writable dir.